src/types.ts

```diff
@@ -10,6 +16,31 @@ export type {
   RuntimeSlackFunctionHandler as SlackFunctionHandler,
 } from "./functions/types.ts";
 
+// ----------------------------------------------------------------------------
+// Runtime Types
+// ----------------------------------------------------------------------------
+
+// deno-lint-ignore no-explicit-any
+type AnyCustomType = CustomType<any, any, any>;
```

Comment:
Rather than passing `any`, we can pass the exact shape that these expect. [We actually already do this for `TypedObjectParameterDefinition` and call it `TypedObjectParameter`](https://github.com/slackapi/deno-slack-sdk/blob/main/src/parameters/definition_types.ts#L98).

```ts
type AnyCustomType = CustomType<
  TypedObjectProperties,
  TypedObjectRequiredProperties<TypedObjectProperties>,
  CustomTypeDefinition<TypedObjectProperties, TypedObjectRequiredProperties<TypedObjectProperties>>
>;

type AnyTypedObjectParameterDefinition = TypedObjectParameter
```

Reply:
:chefs-kiss: